main.js

```diff
@@ -46,6 +48,14 @@ const animate = () => {
   drawPaddles();
   drawBall();
   updatePaddles();
+  ball.x += ballSpeedHorizontal;
+  ball.y += ballSpeedVertical;
+  if (ball.x < 10 || ball.x > 949) {
+    ballSpeedHorizontal *= -1;
+  }
+  if (ball.y < 10 || ball.y > 539) {
```

Comment:
Same comment on the width apply here. Use const for canvas size and calculate offset using the dimensions of the ball.